packages/api/src/types/index.ts

```diff
@@ -73,6 +73,13 @@ export interface GraphQLOptions {
 
 export interface GraphQLResult {
     data?: object,
-    errors?: [object],
+    errors?: [{
+        message: string,
+        locations?: [{
+            line: number,
+            column: number
+        }],
+        path?: [string | number]
```

Comment:
```suggestion
        path?: [string | number],
        extensions?: object 
```

I would add this extensions optional as well. @manueliglesias what do you think?